File: C++/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
      int sum = 0;
      int result = 0;
      for(int i = 0; i < accounts.size(); i++){
        for(int j = 0; j < accounts[i].size(); j++){
          sum += accounts[i][j];
        }
        result = max(result, sum);
        sum = 0;
      }
      return result;
    }
};

File: Python/1672-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        result = 0
        for i in range(0, len(accounts)):
            sum = 0
            for j in range(0, len(accounts[i])):
                sum = sum + accounts[i][j]
            result = max(result, sum)
        return result
